pages/index.js

```diff
@@ -1,11 +1,14 @@
 import Heading from "../components/Heading";
 import SubHeading from "../components/SubHeading";
+import DogList from "../components/DogList";
 
 export default function Home() {
```

Comment:
Your `Home` page should accept the `dogs` prop here, to pass it down to the `DogsList` component, like: 

```
import Heading from "../components/Heading";
import DogList from "../components/DogList";

export default function Home({ dogs }) {
  return (
    <main>
      <Heading>Dog Pass 🐕‍🦺</Heading>
      <DogList dogs={dogs} />
    </main>
  );
}

```

And than the first step, even it's my last comment would be, to declare the mock array `dogs` in the `_app.js ` and pass the prop `dogs` to the components. It could be written like that: 

```
const dogs = [
  {
    name: "Anton",
    dateOfBirth: "14.07.2014",
    placeOfBirth: "Romania",
    sex: "male",
    color: "black",
    breed: "Shepard Labrador Mix",
  },
  {
    name: "Canella",
    dateOfBirth: "01.05.2018",
    placeOfBirth: "Spain",
    sex: "female",
    color: "brown",
    breed: "Unknown Mix",
  },
];

export default function App({ Component, pageProps }) {
  return (
    <>
      <GlobalStyle />
      <Head>
        <title>Dog Pass</title>
      </Head>
      <Component {...pageProps} dogs={dogs} />
    </>
  );
}

```